lib/ansible/modules/cloud/azure/azure_rm_functionapp.py

```diff
@@ -34,6 +34,29 @@
     location:
         description:
             - Valid Azure location. Defaults to location of the resource group.
+    plan:
+        description:
+            - App service plan. Required for creation.
+            - It can be name of existing app service plan in same resource group as web app.
```

Comment:
` as web app` sounds confusing..

Reply:
yes, actually it's not required. if not specified, something will be created anyway by the service.

Reply:
updated the doc